pom.xml

```diff
@@ -26,6 +26,12 @@
             <version>4.12</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter</artifactId>
+            <version>RELEASE</version>
+            <scope>test</scope>
+        </dependency>
```

Comment:
<img width="471" alt="image" src="https://user-images.githubusercontent.com/72727656/199205730-f496f184-edb8-4774-815a-47fe1d722ba4.png">

Reply:
but I use JUnit4

Reply:
maybe something went wrong. I have select JUnit4 when generate test for classes

Reply:
How I can fix that?